portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/upgrades/SAMLUpgrade.testcase

```diff
@@ -31,6 +31,21 @@ definition {
 		Upgrade.viewSAMLSPConnectionConfiguration();
 	}
 
+	@priority = "5"
+	test ViewSAMLConfigurationAfterUpgrade73101 {
+		property data.archive.type = "data-archive-saml";
+		property database.types = "db2,mariadb,mysql,postgresql,sqlserver,sybase";
+		property portal.version = "7.3.10.1";
+
+		Upgrade.viewSAMLGeneralConfiguration(samlRoleType = "Identity Provider");
+
+		Upgrade.viewSAMLSPConnectionConfiguration();
+
+		Upgrade.configureCertificatesAfterUpgrade();
```

Comment:
This macro does not exist in the Upgrade.macro, please check the macro names you added in the previous commit:
https://github.com/gaborlovasdotcom/liferay-portal/pull/133/commits/4401bce9fbcd6b1635717ab4cf016456a7417c65